src/css/cart.css

```diff
@@ -138,15 +133,39 @@
     align-self: flex-start;
     border-radius: 4px;
     box-shadow: var(--shadow-cart);
+    position: sticky;
+    top: 120px;
+    margin: 0 auto;
 }
 
 .divider-line {
     margin: 0.5rem 0 0.5rem 0;
 }
 
+
 .charges-amt{
     display: flex;
     flex-direction: row;
     justify-content: space-between;
     align-items: center;
-}
\ No newline at end of file
+}
+
+
+.checkout{
+    position: sticky;
+    top: 120px;
```

Comment:
Suggestion- Try to be consistent with units.

Reply:
sure, thanks